client/src/components/BottomBanner/index.tsx

```diff
@@ -51,16 +60,9 @@ const BottomBanner = memo(
             dismissible
             sdsType="primary"
             onClose={setBottomBannerLastClosedTime}
-            // @ts-expect-error -- czifui Banner component types text prop as a string but the prop works with JSX as well
-            text={
-              <span>
-                {BOTTOM_BANNER_SURVEY_TEXT}
-                <StyledLink href={surveyLink} target="_blank" rel="noopener">
-                  {BOTTOM_BANNER_SURVEY_LINK_TEXT}
-                </StyledLink>
-              </span>
-            }
-          />
+          >
+            {bannerContent}
```

Comment:
The `bannerContent` is passed as `children` to the `StyledBanner` component instead of using a text prop - this resolves the type error